cves/CVE-2019-9721.yml

```diff
@@ -186,8 +195,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: The result of this vulnerability is hogging processing power and would not plausible to be detected automatically
```

Comment:
There are automated performance regression tools that can work to observe and detect bad performance issues like this.